ramls/item.json

```diff
@@ -51,6 +51,16 @@
       "type": "string",
       "description": "Identifies the source of the call number, e.g., LCC, Dewey, NLM, etc."
     },
+    "effectiveCallNumberComponents": {
```

Comment:
Since this is generated on the backend for every POST/PUT and any value passed in is ignored, can this be marked `readonly`?